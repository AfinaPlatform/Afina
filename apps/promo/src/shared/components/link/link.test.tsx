import Link from "./link";
import { render } from "@testing-library/react";

describe("shared/link", () => {
  it("render a html tag", () => {
    const { container } = render(<Link href="/">Click</Link>);
    expect(container.querySelector("a")?.tagName).toBe("A");
  });

  it("snapshot", () => {
    const { asFragment } = render(<Link href="/">Click</Link>);
    expect(asFragment()).toMatchSnapshot();
  });
});
