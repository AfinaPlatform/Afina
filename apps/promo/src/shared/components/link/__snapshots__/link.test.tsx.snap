// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/link snapshot 1`] = `
<DocumentFragment>
  <a
    href="/"
  >
    Click
  </a>
</DocumentFragment>
`;
